classes/ActionScheduler_Data.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+use Action_Scheduler\Migration\ActionScheduler_MigrationConfig;
+use Action_Scheduler\Migration\ActionScheduler_MigrationScheduler;
+use Action_Scheduler\WP_CLI\ActionScheduler_WPCLI_Migration_Command;
+use Action_Scheduler\WP_CLI\ActionScheduler_WPCLI_ProgressBar;
+
+/**
+ * Class ActionScheduler_Data
+ *
+ * The main plugin/initialization class for the
+ * Action Scheduler Custom Tables plugin.
+ *
+ * Responsible for hooking everything up with WordPress.
+ *
+ * @package Action_Scheduler
+ *
+ * @codeCoverageIgnore
+ */
+class ActionScheduler_Data {
```

Comment:
There's a few methods in this class which would be better suited elsewhere (I've left comments on them individually). Once they're moved, the class seems to be focused primarily on migration, so it may be better off renamed to something conveying that.